.github/workflows/linux.yml

```diff
@@ -58,9 +58,10 @@ jobs:
           ignore_words_list: fo,wee,addin,notin
       - name: Install dependencies
         run: |
-          dnf install -y make gcc-c++ libasan clang-analyzer cmake dnf-plugins-core epel-release
+          dnf update -y
```

Comment:
Only bad package managers require this, DNF isn't one of them.

Reply:
Hm ... that should only happen on amateur distros like Debian (SCNR). DNF notices if there's no index (like in a fresh Docker images) and downloads it. The same after adding repositories. It also updates the index periodically.